src/components/index.ts

```diff
@@ -1,5 +1,7 @@
 export { default as Header } from './Header';
 export { default as Layout } from './Layout';
+
+export { default as SearchBar } from './SearchBar';
```

Comment:
```suggestion
export { default as SearchBar } from './SearchBar';
```

불필요한 공백은 제거해도 좋을 듯 합니다

Reply:
넵 반영하겠습니다!